libs/agno/agno/models/google/gemini.py

```diff
@@ -239,6 +244,13 @@ def get_request_params(
                 Tool(retrieval=Retrieval(vertex_ai_search=VertexAISearch(datastore=self.vertexai_search_datastore)))
             )
 
+        if self.file_search_store_names:
+            log_debug("Gemini File Search enabled.")
+            file_search_config: Dict[str, Any] = {"file_search_store_names": self.file_search_store_names}
+            if self.file_search_metadata_filter:
+                file_search_config["metadata_filter"] = self.file_search_metadata_filter
+            builtin_tools.append(Tool(file_search=FileSearch(**file_search_config)))  # type: ignore[arg-type]
```

Comment:
Can we encapsulate this in a `append_file_search_tool` function, to clarify that's what's happening?